### ISSUE TITLE
Error with ClaimMany action in v0.19.3

### ISSUE THREAD

**Issue by bleiva** [graphprotocol/indexer#435]
version: v0.19.3

I got this error in my logs and I can't claim these allocations for rebate pool:

```
[2022-07-07 13:28:08.865 +0000] WARN (IndexerAgent/1 on indexer-agent-0): Failed to claim allocations
    component: "Network"
    indexer: "0xb06071394531B63b0bac78f27e12dc2BEaA913E4"
    operator: "0x7A361DB89c9419699DEf3349b5F6F1cBA294267D"
    action: "ClaimMany"
    err: {
      "type": "IndexerError",
      "message": "Failed to claim allocation",
      "stack":
          IndexerError: Failed to claim allocation
              at indexerError (/opt/indexer/packages/indexer-common/dist/errors.js:143:12)
              at Network.<anonymous> (/opt/indexer/packages/indexer-common/dist/network.js:993:60)
              at Generator.next (<anonymous>)
              at fulfilled (/opt/indexer/packages/indexer-common/dist/network.js:5:58)
              at runMicrotasks (<anonymous>)
              at processTicksAndRejections (node:internal/process/task_queues:96:5)
      "code": "IE016",
      "explanation": "https://github.com/graphprotocol/indexer/blob/master/docs/errors.md#ie016",
      "cause": {
        "type": "TypeError",
        "message": "_a.gt is not a function",
        "stack":
            TypeError: _a.gt is not a function
                at /opt/indexer/packages/indexer-common/dist/network.js:969:124
                at Array.sort (<anonymous>)
                at Network.<anonymous> (/opt/indexer/packages/indexer-common/dist/network.js:969:22)
                at Generator.next (<anonymous>)
                at fulfilled (/opt/indexer/packages/indexer-common/dist/network.js:5:58)
                at runMicrotasks (<anonymous>)
                at processTicksAndRejections (node:internal/process/task_queues:96:5)
      }
    }
```
And we can see in the logs that there many allocations pending:

```
[2022-07-07 13:38:08.653 +0000] INFO (IndexerAgent/1 on indexer-agent-0): 57 allocations are eligible for rebate pool claims
```

Thanks

**Comment by fordN** [comment]
Hey @bleiva, thanks for reporting.

Could you share the node js version that you are using? I'm trying to reproduce the issue.

**Comment by bleiva** [comment]
We are running the containers inside our Kubernetes Cluster, the nodejs version which is running the container

ghcr.io/graphprotocol/indexer-agent:v0.19.3

```
root@indexer-agent-0:~# node -v
v16.15.1
root@indexer-agent-0:~# 
```

On the other hand, our instances are Amazon Linux 2, beta.kubernetes.io/arch=amd64, beta.kubernetes.io/instance-type=r5a.xlarge

**Comment by vincenttaglia** [comment]
I'm on Ubuntu 20.04.4 LTS and Node v16.16.0 and I'm getting the same error. Using indexer-agent v0.19.3 installed via NPM.

Related logs:
```
{"level":30,"time":1658159085387,"pid":999887,"hostname":"indexer-agent","name":"IndexerAgent","component":"Agent","claimable":[{"id":"0x35738cf0439698CCFB4eC0fDe38EA7511ec1D23E","deployment":{"bytes32":"0xd991edfccdeef8bc78f513a9cf2afeedbae74ab23041a3ebdb33033c0255f731","ipfsHash":"QmcyxEvXAKxs7n1WgKY4sqavvPa7VWfL8KuR7JSnjWjHVz"},"createdAtEpoch":539},{"id":"0x9c5e5911Bd56BFFD709232A796918D3fBF5Bd504","deployment":{"bytes32":"0x8e01ba7793e57b4f9a4e3d6c0c69e30f06934731297747d9030e1adac3eb577e","ipfsHash":"QmXtz8Zf837khnHFX1DbPY8cv97fSNd25T19Fjgv4yekGd"},"createdAtEpoch":535},{"id":"0xbAB4406315461fe1D49F7EE596f7DCc80BF64b53","deployment":{"bytes32":"0x2b4f18b8b904abe5fe6dc93e4f42c38e9d7a037b8a765345c128dfd6d2c8e6e3","ipfsHash":"QmRFiCChCVwbSJ7VuEYiesEb5wHo89CuKyYS2digs9mZRL"},"createdAtEpoch":516},{"id":"0xCA8150f09c82Db28672bf218e480F57D5Bf207B6","deployment":{"bytes32":"0x1cbe978fdda55597c52276bbbcd5492b6b1746ff8d129efb5477d4aa9d53650e","ipfsHash":"QmQGrgikMWew2Rt2oD88vjCHAe9KEzZkpujUJAHmw834KF"},"createdAtEpoch":524}],"msg":"Claim rebate rewards"}
{"level":30,"time":1658159085388,"pid":999887,"hostname":"indexer-agent","name":"IndexerAgent","component":"Network","indexer":"0x87Eba079059B75504c734820d6cf828476754B83","operator":"0xC56961836857210e256D71c91a62e90865075380","action":"ClaimMany","allocations":[{"allocation":"0x35738cf0439698CCFB4eC0fDe38EA7511ec1D23E","deployment":{"bytes32":"0xd991edfccdeef8bc78f513a9cf2afeedbae74ab23041a3ebdb33033c0255f731","ipfsHash":"QmcyxEvXAKxs7n1WgKY4sqavvPa7VWfL8KuR7JSnjWjHVz"},"createdAtEpoch":539,"closedAtEpoch":542,"createdAtBlockHash":"0xeda9e9266dc4781365ecf70d4ffdf6e253221400ac70c7300fc29ba9847d985b"},{"allocation":"0x9c5e5911Bd56BFFD709232A796918D3fBF5Bd504","deployment":{"bytes32":"0x8e01ba7793e57b4f9a4e3d6c0c69e30f06934731297747d9030e1adac3eb577e","ipfsHash":"QmXtz8Zf837khnHFX1DbPY8cv97fSNd25T19Fjgv4yekGd"},"createdAtEpoch":535,"closedAtEpoch":549,"createdAtBlockHash":"0xd4b8c212b7d09a62c5a754bc7612237930d8c296472a2e4d64f190f21ad4c01a"},{"allocation":"0xbAB4406315461fe1D49F7EE596f7DCc80BF64b53","deployment":{"bytes32":"0x2b4f18b8b904abe5fe6dc93e4f42c38e9d7a037b8a765345c128dfd6d2c8e6e3","ipfsHash":"QmRFiCChCVwbSJ7VuEYiesEb5wHo89CuKyYS2digs9mZRL"},"createdAtEpoch":516,"closedAtEpoch":543,"createdAtBlockHash":"0x7a94d687ef76256d86a24cfdb878a2ee947e192270172b747b7a0947e58830ce"},{"allocation":"0xCA8150f09c82Db28672bf218e480F57D5Bf207B6","deployment":{"bytes32":"0x1cbe978fdda55597c52276bbbcd5492b6b1746ff8d129efb5477d4aa9d53650e","ipfsHash":"QmQGrgikMWew2Rt2oD88vjCHAe9KEzZkpujUJAHmw834KF"},"createdAtEpoch":524,"closedAtEpoch":542,"createdAtBlockHash":"0x08d47dcd5fabbdef00a621c61d7bc5d9dd32f3e1f16c26501d3695aab0bfb70d"}],"restakeRewards":false,"msg":"4 allocations are eligible for rebate pool claims"}
{"level":40,"time":1658159085626,"pid":999887,"hostname":"indexer-agent","name":"IndexerAgent","component":"Network","indexer":"0x87Eba079059B75504c734820d6cf828476754B83","operator":"0xC56961836857210e256D71c91a62e90865075380","action":"ClaimMany","err":{"type":"IndexerError","message":"Failed to claim allocation","stack":"IndexerError: Failed to claim allocation\n    at indexerError (/home/ubuntu/.nvm/versions/node/v14.20.0/lib/node_modules/@graphprotocol/indexer-agent/node_modules/@graphprotocol/indexer-common/dist/errors.js:143:12)\n    at Network.<anonymous> (/home/ubuntu/.nvm/versions/node/v14.20.0/lib/node_modules/@graphprotocol/indexer-agent/node_modules/@graphprotocol/indexer-common/dist/network.js:993:60)\n    at Generator.next (<anonymous>)\n    at fulfilled (/home/ubuntu/.nvm/versions/node/v14.20.0/lib/node_modules/@graphprotocol/indexer-agent/node_modules/@graphprotocol/indexer-common/dist/network.js:5:58)\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (internal/process/task_queues.js:95:5)","code":"IE016","explanation":"https://github.com/graphprotocol/indexer/blob/master/docs/errors.md#ie016","cause":{"type":"TypeError","message":"_a.gt is not a function","stack":"TypeError: _a.gt is not a function\n    at /home/ubuntu/.nvm/versions/node/v14.20.0/lib/node_modules/@graphprotocol/indexer-agent/node_modules/@graphprotocol/indexer-common/dist/network.js:969:124\n    at Array.sort (<anonymous>)\n    at Network.<anonymous> (/home/ubuntu/.nvm/versions/node/v14.20.0/lib/node_modules/@graphprotocol/indexer-agent/node_modules/@graphprotocol/indexer-common/dist/network.js:969:22)\n    at Generator.next (<anonymous>)\n    at fulfilled (/home/ubuntu/.nvm/versions/node/v14.20.0/lib/node_modules/@graphprotocol/indexer-agent/node_modules/@graphprotocol/indexer-common/dist/network.js:5:58)\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (internal/process/task_queues.js:95:5)"}},"msg":"Failed to claim allocations"}
```

**Comment by chriswessels** [comment]
@hopeyen wrote this PR which *should* fix it https://github.com/graphprotocol/indexer/pull/437

**Comment by fordN** [comment]
Resolved in #437
